src/stores/room-invitation-store.js

```diff
@@ -8,31 +7,35 @@ const roomInvitationMetadataProjection = {
   expires: 1
 };
 
-class RoomInvitationStore extends StoreBase {
+class RoomInvitationStore {
```

Comment:
Good idea to not extend `StoreBase` anymore 👍

Reply:
Yeah, we can throw it away after we've updated all stores.